src/components/themed/WalletList.js

```diff
@@ -79,10 +79,11 @@ type Props = OwnProps & StateProps & DispatchProps & ThemeProps
 
 class WalletListComponent extends React.PureComponent<Props> {
   sortWalletList(walletList: WalletListItem[]): WalletListItem[] {
-    const getFiatBalance = (wallet: GuiWallet, fullCurrencyCode: string) => {
+    const getFiatBalance = (wallet: GuiWallet, fullCurrencyCode: string): number => {
       const { settings, exchangeRates } = this.props
       const currencyCode = getSortOptionsCurrencyCode(fullCurrencyCode)
-      return calculateWalletFiatBalanceUsingDefaultIsoFiat(wallet, currencyCode, settings, exchangeRates)
+      const fiatBalanceString = calculateWalletFiatBalanceUsingDefaultIsoFiat(wallet, currencyCode, settings, exchangeRates)
+      return parseFloat(fiatBalanceString)
```

Comment:
No need to convert back to number.
Instead, when you use this function to compare balances, just use the bns comparison functions: 

https://www.npmjs.com/package/biggystring#comparisons

Reply:
It's not use for comparison but for the Array.sort function so it is needed to have a numerical result.